Auth0Tests/Responses.swift

```diff
@@ -115,3 +118,22 @@ func managementResponse(_ payload: Any) -> OHHTTPStubsResponse {
 func managementErrorResponse(error: String, description: String, code: String, statusCode: Int = 400) -> OHHTTPStubsResponse {
     return OHHTTPStubsResponse(jsonObject: ["code": code, "description": description, "statusCode": statusCode, "error": error], statusCode: Int32(statusCode), headers: ["Content-Type": "application/json"])
 }
+
+func jwksResponse(kid: String? = JWKKid) -> OHHTTPStubsResponse {
+    let jwk = generateRSAJWK()
+    let jwks = ["keys": [
```

Comment:
nit, inconsistent format

Reply:
Fixed in c99945de2ec0aead4b9c5c9993162de5e1f08dfa.